src/Clustering/GMUClusterManager.m

```diff
@@ -50,16 +50,6 @@ @implementation GMUClusterManager {
   id<GMUClusterRenderer> _renderer;
 }
 
-// Disables init.
-- (instancetype)init {
```

Comment:
Since we've removed default init method, can we also remove the initWithNullableMap and only keep 
```
- (instancetype)initWithMap:(GMSMapView *)mapView
                  algorithm:(id<GMUClusterAlgorithm>)algorithm
                   renderer:(id<GMUClusterRenderer>)renderer
```